### PR TITLE
Added the path.expand for the bin - needed for if you do 

### PR DESCRIPTION
**Pull request by muschellij2**
This should now allow for: `rdom(filename = "~/tilde_path")`, which failed previously.

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -14,3 +14,7 @@ find_phantom <- function() {
 "%||%" <- function(x, y = NULL) {
   if (missing(x)) y else x
 }
+
+"%pe%" <- function(x, y = "") {
+  if (missing(x)) y else path.expand(x)
+}
```
Comment (cpsievert) [inline R/utils.R:20]: perhaps this should be `normalizePath()` so the user gets an adequate warning if the path doesn't exist?

Comment (cpsievert) [reply R/utils.R:20]: whoops, nevermind, forgot this arg is for writing to a file

**Comment by cpsievert** [comment]
Thanks!

**Comment by muschellij2** [comment]
Probably better.

John

**Pull request merged**